### PR TITLE
added bash script to cloen repo from github

### PR DESCRIPTION
**Pull request by smolamarcin**

### CODE REVIEW

File: runScript [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#! /bin/bash
+sett -e
```
Comment (rivalez) [inline runScript]: misspelling

File: runScript [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+#
+echo Hello Tomasz. Its our demo. Please enter a directory name:
+read foldername
+git clone https://github.com/smolamarcin/battleship $foldername
+cd "$foldername"
+mvn clean install;
```
Comment (rivalez) [inline runScript]: echo: Invoking mvn clean install command

File: runScript [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+git remote show origin | grep tracked | wc -l
+ 
+#Number of commits per author
+echo Number of commits per author:
+git shortlog | grep -E '^[^ ]' 
+
+#count of lines per author
+echo Number of lines per author 
+git ls-files | while read f; do git blame --line-porcelain $f | grep '^author '; done | sort -f | uniq -ic | sort -n 
+#Count of tests
```
Comment (rivalez) [inline runScript]: delete

File: runScript [hunk 4]
```diff
@@ -0,0 +1,34 @@
+#! /bin/bash
+sett -e
+#
+echo Hello Tomasz. Its our demo. Please enter a directory name:
```
Comment (EmiliaCiastek) [inline runScript]: ' is missing: "It's"

File: runScript [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+read foldername
+git clone https://github.com/smolamarcin/battleship $foldername
+cd "$foldername"
+mvn clean install;
+mvn test;
+
+#checkstyle
+mvn checkstyle:checkstyle;
+
```
Comment (EmiliaCiastek) [inline runScript]: finbugs and jacoco reports are missing

File: runScript (hunk 3 above)
Comment (EmiliaCiastek) [reply runScript]: I think that all counts are unnecessary

File: runScript (hunk 5 above)
Comment (smolamarcin) [reply runScript]: Findbugs and jacoco will be added in next release.

**Pull request merged**